parser.js

```diff
@@ -73,6 +87,15 @@ const metadataRules = {
       ['link[rel="Shortcut Icon"]', node => node.element.getAttribute('href')],
       ['link[rel="mask-icon"]', node => node.element.getAttribute('href')],
     ],
+    scorers: [
+      (node, score) => {
+        const sizes = node.element.getAttribute('sizes');
+
+        if (sizes) {
+          return score + sizes.match(/\d+/g).reduce((a, b) => a * b);
+        }
+      }
```

Comment:
Do we need an `else` here or anything? (ie: if `sizes` is falsey, do we return original `score` or whatever?)